### PR TITLE
Fix back blast if damage not allowed

### PR DESCRIPTION
**Pull request by Dystopian**

### CODE REVIEW

**Comment by PabstMirror** [comment]
good catch

File: addons/overpressure/functions/fnc_firedEHBB.sqf [hunk 1]
```diff
@@ -54,8 +54,8 @@ if (_distance < _backblastRange) then {
     if (isClass (configFile >> "CfgPatches" >> "ACE_Medical") && {([_unit] call EFUNC(medical,hasMedicalEnabled))}) then {
         [_unit, _damage, "body", "backblast"] call EFUNC(medical,addDamageToUnit);
     } else {
-        TRACE_1("",isDamageAllowed _x);
-        if (!isDamageAllowed _x) exitWith {}; // Skip damage if not allowed
+        TRACE_1("",isDamageAllowed _unit);
+        if (!isDamageAllowed _unit) exitWith {}; // Skip damage if not allowed
         _unit setDamage (damage _unit + _damage);
```
Comment (commy2) [inline addons/overpressure/functions/fnc_firedEHBB.sqf]: Any reason this is an `exitWith` when it could be a `then` for simplicity?

Comment (Dystopian) [reply addons/overpressure/functions/fnc_firedEHBB.sqf]: I like `exitWith` :)

Comment (commy2) [reply addons/overpressure/functions/fnc_firedEHBB.sqf]: Same. But I like classical control structures even more.

Comment (Dystopian) [reply addons/overpressure/functions/fnc_firedEHBB.sqf]: classical one makes too many indents

Comment (PabstMirror) [reply addons/overpressure/functions/fnc_firedEHBB.sqf]: I think I might have just been trying to minimize the git diff, which isn't really a good reason

**Pull request merged**